diagnostic.md

```diff
@@ -54,7 +54,7 @@ so much for us?
 // your response here
 ```
 
-What does typing `rails console` in a directory that houses a rails app do for
+What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
```

Comment:
In fact, I did use `bin/rails db` but elected to stay away from `bin/rails console` until `rails-api-one-to-many`.

I'm not sure that's the way it should be. But it's the way it was done.

If both need to be covered, they should have learning objectives.